__tests__/app.test.js

```diff
@@ -73,4 +73,94 @@ describe('api', ()=>{
             })
         })
     })
+    describe('GET /api/articles', ()=>{
+        test('should return statuscode 200 and  an array of article objects', ()=>{
+            return request(app).get('/api/articles')
+            .expect(200)
+            .then(({body})=>{
+                const {articles} = body;
+                articles.forEach((article)=>{
+                expect(typeof article['article_id']).toBe('number');
+                expect(typeof article['title']).toBe('string');
+                expect(typeof article['topic']).toBe('string');
+                expect(typeof article['author']).toBe('string');
+                expect(typeof article['created_at']).toBe("string");
+                expect(typeof article['votes']).toBe('number');
+                expect(typeof article['article_img_url']).toBe('string');
+                expect(typeof article['comment_count']).toBe('number')
+                })
```

Comment:
I think this is ok, I would look into different matchers that would make it look nicer though. Maybe have a look at .toMatchObject()